cypress/e2e/wipWallet/demoResetBalance.cy.js

```diff
@@ -1,13 +1,14 @@
 import '@testing-library/cypress/add-commands'
 
 function reset_balance_demo() {
-  cy.findByText('Demo').scrollIntoView()
-  cy.get('[class*="virtual"].wallets-accordion__header--virtual')
-    .find('.wallets-accordion__dropdown > svg')
-    .click()
-  cy.get('[class*="virtual"].wallets-accordion__header--virtual')
-    .contains('Reset balance')
-    .click()
+  cy.get('.wallets-dropdown__button').click()
+  cy.findByText('USD Demo Wallet').scrollIntoView()
+  cy.get('.wallets-list-card-dropdown__item-content').contains('USD Demo Wallet').click()
+  cy.get('.wallets-list-details__content').within(()=>{
+    cy.contains('USD').should('be.visible')
+  })
+  cy.contains('Reset balance').should('be.visible')
+  cy.contains('Reset balance').click()
```

Comment:
can you please use Testing library locators here 
```suggestion
    cy.findByText('USD').should('be.visible')
  })
  cy.findByText('Reset balance').should('be.visible')
  cy.findByText('Reset balance').click()
```

Reply:
Updated